test/cli/createkey.py

```diff
@@ -11,25 +11,34 @@
 
 import _utils_
 
-supported_keyspec = ["EH_AES_GCM_128", "EH_RSA_3072"]
+supported_keyspec = ["EH_AES_GCM_128", "EH_AES_GCM_192", "EH_AES_GCM_256", "EH_RSA_2048", "EH_RSA_3072", "EH_RSA_4096", "EH_RSA_3072", "EH_EC_P256", "EH_SM2", "EH_SM4"]
 
 def get_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('--url', type=str, help='the address of the ehsm_kms_server', required=True)
     parser.add_argument('--keyspec', type=str, help='the keyspec [EH_AES_GCM_128, EH_RSA_3072]', required=True)
     parser.add_argument('--origin', type=str, help='the key origin [default is: EH_INTERNAL_KEY]')
+    parser.add_argument('--purpose', type=str, help='the key purpose')
+    parser.add_argument('--padding_mode', type=str, help='the key padding_mode')
+    parser.add_argument('--digest_mode', type=str, help='the key digest_mode')
     args = parser.parse_args()
 
     base_url = args.url + "/ehsm?Action="
     print(base_url)
-    return base_url, args.keyspec, args.origin
+    return base_url, args.keyspec, args.origin, args.purpose, args.padding_mode, args.digest_mode
 
-def createkey(base_url, keyspec, origin):
+def createkey(base_url, keyspec, origin, purpose, padding_mode, digest_mode):
```

Comment:
if not a must param, use "purpose=None" codes to set default values.

Reply:
it will be changed